src/utils/functionUtils.js

```diff
@@ -0,0 +1,384 @@
+      detailData[dataType]?.changes[0] !== null
+    ) {
+      return (
+        <>
+          <Body color={changeColor(dataType, 'INSERT')}>
+            <a
+              href={handleClickLink(
+                detailData[dataType]?.changes[0] ||
+                  detailData[dataType]?.original,
+              )}
+              target="_blank"
+              rel="noreferrer noopener">
+              {detailData[dataType].changes[0]
+                ? detailData[dataType].changes[0]
+                : '---'}
+              {(detailData[dataType].changes[0] && (
+                <LinkIcon height="15" width="30" />
+              )) ||
+                (detailData[dataType].original && (
+                  <LinkIcon height="15" width="30" />
+                ))}
+            </a>
+          </Body>
+          {detailData[dataType]?.changes[0] && detailData[dataType]?.original && (
+            <Body color={changeColor(dataType, 'DELETE')}>
+              <s>
+                <a
+                  href={handleClickLink(detailData[dataType]?.original)}
+                  target="_blank"
+                  rel="noreferrer noopener">
+                  {detailData[dataType].original
+                    ? detailData[dataType].original
+                    : '---'}
+                  {detailData[dataType].original && (
+                    <LinkIcon height="15" width="30" />
+                  )}
+                </a>
+              </s>
+            </Body>
+          )}
+        </>
+      );
+    } else if (
+      detailData[dataType]?.changes[0] === '' &&
+      detailData[dataType].original
+    ) {
+      //subform links that are not deleted
+      return (
+        <Body color={changeColor(dataType, 'INSERT')}>
+          <a
+            href={handleClickLink(detailData[dataType]?.original)}
+            target="_blank"
+            rel="noreferrer noopener">
+            {detailData[dataType]?.original
+              ? detailData[dataType].original
+              : '---'}
+            {detailData[dataType]?.original && (
+              <LinkIcon height="15" width="30" />
+            )}
+          </a>
+        </Body>
+      );
+    } else if (_.isEmpty(detailData[dataType]?.changes)) {
+      //show new subforms links in staging detail view
+      return (
+        <Body color={changeColor(dataType, 'INSERT')}>
+          <a
+            href={handleClickLink(detailData[dataType]?.original)}
+            target="_blank"
+            rel="noreferrer noopener">
+            {detailData[dataType]?.original
+              ? detailData[dataType].original
+              : '---'}
+            {detailData[dataType]?.original && (
+              <LinkIcon height="15" width="30" />
+            )}
+          </a>
+        </Body>
+      );
+    } else if (detailData[dataType]?.changes[0] === null) {
+      // show deleted subform links
+      return (
+        <Body color={changeColor(dataType, 'DELETE')}>
+          <s>
```

Comment:
what is `<s>`?

Reply:
Those are the lines you see across the red